docs/dashboard/configuration/shipping.md

```diff
@@ -53,17 +53,17 @@ These two methods cover most of the basic use cases, for example:
 * Increasing the shipping price for heavy orders.
 
 
-### ZIP Codes
+### Postal codes
 
-This section allows you to exclude shipping method for customers with certain ZIP codes.
-If customer shipping address ZIP Code is within any of specified ranges, that shipping method will be unavailable for that customer.
+This section allows you to exclude shipping method for customers with certain postal codes.
+If customer shipping address postal code is within any of specified ranges, that shipping method will be unavailable for that customer.
 
```

Comment:
If a customer's shipping address is within any of the defined postal code ranges, that shipping method will be unavailable for that customer.